js/main.js

```diff
@@ -932,7 +933,14 @@ var mapView = {
           if (a.creation_time < b.creation_time) return 1;
           return 0;
         });
-        break;
+        break;      
+      case 'level':
```

Comment:
`case 'level'` is wrong and sould be `case 'lvl'` as we switch on the value of the "data-sort"-attribute.